### DISCUSSION TITLE
科研选项过滤器

### DISCUSSION THREAD

**Discussion by daichuanqi11** [LmeSzinc/AzurLaneAutoScript#1084] [Q&A] [answered]
想知道具体到某个船的格式，比方说我想要提高腓特烈大帝的优先级，这一条应该改写成“ Friedrich-0.5 > Friedrich-2.5 > ··· "或者 "Friedrich0.5 > Friedrich2.5 ···" ，还是直接不用写时间 ”Friedrich >  S4-DR0.5 >···“

**Comment by LmeSzinc** [comment, marked as answer]
要用面向对象的方法去思考。每个科研项目都有期数、类型、时长等的属性，过滤器的作用是筛选含有特定属性的对象，没有被限定的属性 Alas 就随便选了。`Friedrich0.5` 的意思是筛选出 产出图纸=Friedrich，时长=0.5 的科研项目，`Friedrich` 的意思是筛选出 产出图纸=Friedrich 的科研项目，因为时长没有限制所以是任意时长的大帝科研。Friedrich0.5 比 Friedrich8 的时均收益高快10倍了，所以一般是分开写。

**Reply by daichuanqi11** [reply]
感谢！
